pkg/virt-launcher/virtwrap/converter/compute/graphics.go

```diff
@@ -55,3 +69,34 @@ func (g GraphicsDomainConfigurator) configureVNCSocket(vmi *v1.VirtualMachineIns
 		},
 	}
 }
+
+func (g GraphicsDomainConfigurator) configureVideoDevice(vmi *v1.VirtualMachineInstance, domain *api.Domain) {
+	switch g.architecture {
+	case "amd64":
+		g.configureAMD64VideoDevice(vmi, domain)
+	}
+}
+
+func (g GraphicsDomainConfigurator) configureAMD64VideoDevice(vmi *v1.VirtualMachineInstance, domain *api.Domain) {
+	// For AMD64 + EFI, use bochs. For BIOS, use VGA
+	if vmi.IsBootloaderEFI() && g.bochsForEFIGuests {
```

Comment:
The original code evaluated these conditions in reverse order which prevented a redundant function call in case `bochsForEFIGuests` is `false`.